routes/runtime/github.rb

```diff
@@ -6,7 +6,11 @@ class Clover
       fail CloverError.new(400, "InvalidRequest", "invalid JWT format or claim in Authorization header")
     end
 
-    repository.setup_blob_storage unless repository.access_key
+    begin
+      repository.setup_blob_storage unless repository.access_key
+    rescue Excon::Error::HTTPStatus
+      fail CloverError.new(400, "InvalidRequest", "unable to setup blob storage")
```

Comment:
Can we log the response body to see the error details?

Reply:
Pushed a commit for that.